### PR TITLE
Review README.md

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-<TODO>

### What?

I have added/removed/altered:

- [x] Keep the README.md file up-to-date

### Why?

I am doing this because:

- The talent team are being replaced by MoJ permanent staff and they won't know what's in all of our heads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,23 +141,26 @@ Docker image, pushing code staging:
 
 This application is deployed to [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/).
 
-Currently we have a `staging` and `production` environment on Cloud
+Currently we have a `dev`, `staging`, `uat`, `preprod` and `production` environment on Cloud
```
Comment (willfish) [inline README.md:144]:
```suggestion
Currently we have a `dev`, `staging`, `uat`, `preprod` and `production` environments on Cloud
```

File: README.md [hunk 2]
```diff
@@ -155,23 +141,26 @@ Docker image, pushing code staging:
...
 
-`production` can be deployed on any successful build of `master` but
-requires manual intervention to approve the `hold_production` step in
-the workflow. To approve a particular deploy for build log into
-CircleCI, find the build that you want to deploy and click the
-`test-build-deploy` link. This should take you to the workflow graph for
-that build where you can click the `hold_production` step to kick off
-the production deploy.
+`preprod`, `uat` and `production` can be deployed by generating a new tag
+following [semantic versioning](https://semver.org/) pointing to the current commit.
```
Comment (willfish) [inline README.md:157]:
```suggestion
following [semantic versioning](https://semver.org/) pointing to the current commit. We typically do this locally with:

```bash
# Get a list of tags
git tag
# Tag following semantic versioning rules
git tag v1.0.1
# Push the tag to origin
git push origin v1.0.1
```

```

File: README.md [hunk 3]
```diff
@@ -155,23 +141,26 @@ Docker image, pushing code staging:
...
 
+Tagged deploys are gated for the `production` environment and require an approval. This is typically done after
+a review from a product owner where reasonable and if a hotfix is not necessary/
```
Comment (willfish) [inline README.md:160]:
```suggestion
a review from a product owner where reasonable and if a hotfix is not necessary.
```

File: README.md [hunk 4]
```diff
@@ -155,23 +141,26 @@ Docker image, pushing code staging:
...
+the workflow.
```
Comment (willfish) [inline README.md:161]: Remove me

**Review by mafonso — APPROVED** [review]
LGTM

**Pull request merged**